packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItemWrapper/ContextualMenuButton.tsx

```diff
@@ -34,17 +35,17 @@ export class ContextualMenuButton extends ContextualMenuItemWrapper {
     const itemHasSubmenu = hasSubmenu(item);
     const { itemProps, ariaLabel } = item;
 
-    const buttonNativeProperties = getNativeProps(item, buttonProperties);
+    const buttonNativeProperties = getNativeProps<React.ButtonHTMLAttributes<HTMLButtonElement>>(item, buttonProperties);
```

Comment:
Hmmmmm, I think this is revealing a gap in `properties.ts`. If `getNativeProps` was modified to use `keyof T` and some other magic and `buttonProperties` were typed as a set of keys of `React.ButtonHTMLAttributes`, maybe we could get a lot of increased type safety without having to explicitly cast here.

Reply:
That would be an awesome change!